source/config-file/specify-connection-options.txt

```diff
@@ -0,0 +1,78 @@
+.. _compass-specify-connection-options:
+
+================================
+Specify Read Preference and Tags
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can pre-assign connection option values for when you connect to your MongoDB 
+deployment through |compass-short| with the ``forceConnectionOptions`` option. 
+This lets you restrict the ways that users can connect to your deployment so
+their read operations do not impact the availability of replica set primaries.
+
+Procedure
+---------
+
+In this example, you set the read preference to ``SECONDARY`` and target members 
+with the ``ANALYTICS`` and ``READ_ONLY`` tags.  You can specify these options in 
+either a command line option or :ref:`configuration file <config-file>` option. 
+
+Command Line Example
+~~~~~~~~~~~~~~~~~~~~
+
+The following command starts |compass| from the command line and uses
+``--forceConnectionOptions`` to set the ``readPreference`` and 
+``readPreferenceTags`` options:
+
+.. code-block:: sh
+
+   <path-to-Compass-executable> --forceConnectionOptions.readPreference=secondary --forceConnectionOptions.readPreferenceTags=nodeType:ANALYTICS --forceConnectionOptions.readPreferenceTags=nodeType:READ_ONLY
```

Comment:
This is hard to read because of the length. I don't know the conventions for Compass docs, but consider a format like:

```
<path-to-Compass-executable> \
--forceConnectionOptions.readPreference=secondary \
--forceConnectionOptions.readPreferenceTags=nodeType:ANALYTICS \
--forceConnectionOptions.readPreferenceTags=nodeType:READ_ONLY
```

Reply:
Ah great point — noted and updated.